sdf/1.10/joint.sdf

```diff
@@ -96,6 +96,23 @@
       </element>
 
     </element> <!-- End Limit -->
+
+    <element name="mimic" required="0">
+      <description>Specifies the joint to be mimicked</description>
+      <attribute name="joint" type="string" default="" required="1">
+        <description>Name of the joint to be mimicked</description>
+      </attribute>
+      <attribute name="axis" type="string" default="1" required="1">
+        <description>Axis number of the joint to be mimicked, can be
+        either 1 or 2</description>
+      </attribute>
+      <element name="multiplier" type="double" default="0" required="1">
```

Comment:
```suggestion
      <element name="multiplier" type="double" default="1.0" required="1">
```

a default value of 0 would lock the joint

Reply:
Done https://github.com/gazebosim/sdformat/pull/1166/commits/cb29c9128868d42aa45686968cd6107de6c5ac76